SlavaJSFE/nodejs-in-aws-2022Q3-3_backend
# feat: complete task 8

### What was done:

- [x] Task 8.1
- [x] Task 8.2
- [x] Task 8.3
- [x] Task 8.4
- [x] +1 (All languages) - Integrate Cart service with FE repository

### Frontend:
- [Air Store](https://d3c5hfqlpim7sq.cloudfront.net)
- [PR for task 8 in Frontend repository](https://github.com/SlavaJSFE/nodejs-in-aws-2022Q3-3_frontend-react/pull/6)

### How to check the task:
1. Open [Frontend Application](https://d3c5hfqlpim7sq.cloudfront.net)
2. In browser console you'll see 401 (Unauthorized) Error for cart endpoint.
3. Set the authorization_token through the console:
```localStorage.setItem('authorization_token', btoa('SlavaJSFE:TEST_PASSWORD'))```
4. In the [cart](https://d3c5hfqlpim7sq.cloudfront.net/cart) you'll see two products:
- Air from Norway (count 2)
- Air from South Africa (count 3)
5. You can change the count in the cart using ```+``` and ```-```
6. You can set another authorization_token with new user:
```localStorage.setItem('authorization_token', btoa('[any new user name]:TEST_PASSWORD'))```

### Database screenshots
![image](https://user-images.githubusercontent.com/70973590/204661602-2b7be92a-0825-4dd9-a785-87e94b21fbaf.png)

![image](https://user-images.githubusercontent.com/70973590/204663129-dcb0daea-5dd7-462d-9069-47423d6759ca.png)


### Self Evaluation: 5
